solutions/security/get-started/get-started-detect-with-siem.md

```diff
@@ -0,0 +1,135 @@
+5. Click **Add the integration**, then Confirm incoming data on the next page.  
+    After a few minutes, you should see the agent receiving data.
+
+    :::{image} /solutions/images/security-gs-siem-install-agent.png
+    :alt: Alerts page with visualizations section collapsed
+    :screenshot:
+    :::
+
+:::{important}
+If you’re using macOS, some versions may require you to grant {{elastic-endpoint}} Full Disk Access to different kernels, system extensions, or files. Refer to [Elastic Defend requirements](/solutions/security/configure-elastic-defend/elastic-defend-requirements.md) for more information.
+:::
+::::
+
+::::{step} Add Elastic detection prebuilt rules
+
+Detection rules allow you to proactively monitor your environment by searching for source events, matches, sequences, or machine learning job anomaly results that meet their criteria. When a rule’s criteria are met, {{elastic-sec}} generates an alert. While you can create your own rules tailored for your environment, Elastic ships out-of-the-box prebuilt rules that you can install. 
+
+To install and enable Elastic's prebuilt detection rules: 
+1. On the Get Started page, scroll down to the **Configure rules and alerts** section. 
+2. At the top of the page, click **Add Elastic rules**. The badge next to it shows the number of prebuilt rules available for installation. 
+
+    The next pages displays the list of rules. Click on a rule name to view its details before you install it.  
+3. Use the search bar and **Tags** filter to find the rules you want to install. For example, to filter by operating system, search for the appropriate OS from the **Tags** menu (such as `macOS`). We recommend installing all the rules for your operating system, but you can install whichever rules you're comfortable with to start. You can always install more later.  
+4. Select the check box next to rules you want to install. To select all rules on the page, select the check box to the left of the **Rule** column heading. 
+5. Click ![Vertical boxes button](/solutions/images/serverless-boxesVertical.svg "") → **Install and enable** to install and start running the rules. Once you enable a rule, it starts running on its configured schedule.
+
+:::{image} /solutions/images/security-gs-siem-install-rules.png 
+:alt: Alerts page with visualizations section collapsed
+:screenshot:
+:::
+    
+    To learn how to view and manage all detection rules, refer to [Manage detection rules](/solutions/security/detect-and-alert/manage-detection-rules.md). 
+::::
+
+::::{step} Visualize and examine alert details 
+
+Now that you've installed and enabled rules, it's time to monitor the {{security-app}} to see if you receive any alerts. Remember, an alert is generated if any of the rule's criteria are met. {{elastic-sec}} provides several tools for investigating security events:
```

Comment:
I can't find our guidelines on using app vs project when describing a solution (i.e., a project) in Serverless. Still flagging this in case our guidance advises against using "app" when describing Serverless functionality. 

```suggestion
Now that you've installed and enabled rules, it's time to monitor your {{sec-serverless}} project to see if you receive any alerts. Remember, an alert is generated if any of the rule's criteria are met. {{elastic-sec}} provides several tools for investigating security events:
```